frontend/src/components/user-administration/user-administration-modal/show-password-modal.tsx

```diff
@@ -0,0 +1,108 @@
+import {
+  Box,
+  Button,
+  ClickAwayListener,
+  Grid,
+  Modal,
+  Tooltip,
+  Typography
+} from '@mui/material';
+import React from 'react';
+import { Icon } from '@iconify/react';
+
+const ShowPasswordModal = ({
+  showModal = true,
+  setShowModal,
+  password
+}: {
+  showModal: boolean;
+  setShowModal: (value: boolean) => void;
+  password: string;
+}) => {
+  const [open, setOpen] = React.useState(false);
+
+  const handleTooltipClose = () => {
+    setOpen(false);
+  };
+
+  const handleTooltipOpen = async () => {
+    navigator.clipboard.writeText(password);
+    setOpen(true);
+    await delay(5000);
+    setOpen(false);
+  };
+
+  return (
+    <div>
+      <Modal open={showModal}>
+        <Box sx={style}>
+          <Typography align="center" component="h1" variant="h5">
+            Initialpasswort
+          </Typography>
+
+          <Grid
+            container
+            justifyContent="center"
+            columnGap={'10px'}
+            style={{ marginTop: '20px' }}
+          >
+            {password}
+
+            <ClickAwayListener onClickAway={handleTooltipClose}>
+              <div>
+                <Tooltip
+                  PopperProps={{
+                    disablePortal: true
+                  }}
+                  onClose={handleTooltipClose}
+                  open={open}
+                  disableFocusListener
+                  disableHoverListener
+                  disableTouchListener
+                  title="Kopiert!"
+                >
+                  <Icon
+                    icon="ic:outline-content-copy"
+                    onClick={handleTooltipOpen}
+                  />
+                </Tooltip>
+              </div>
+            </ClickAwayListener>
```

Comment:
Instead of using the Tooltip for showing a confirmation, we used everywhere else the "toast"

so just execute a "toast.success("This is the message")" on the successfull adding the user

Reply:
But here, the user needs to interact with the UI (copy pw)

I think the toast is more for (additional) user information and closes automatically after some time.

I think an initial pw ist too crucial for a toast, don't you think?